pipeline-scripts/delete-inactive-guest-users.sh

```diff
@@ -17,9 +20,27 @@ users_file=guests.json
 delete_inactive_guests() {
 
   # Create file with list of guest users that have accepted their invite
-  az rest --method GET --uri "https://graph.microsoft.com/beta/users?\$top=999&\$filter=externalUserState eq 'Accepted' and userType eq 'Guest' and createdDateTime le ${min_user_age_date}&\$select=id,displayName,signInActivity,createdDateTime,mail" > ${users_file}
+  NEXT_LINK=
+  counter=0
+
+  until [[ "${NEXT_LINK}" == "null" ]]; do
+
+    if [[ ${counter} == 0 ]]; then
+      az rest --method get --uri "https://graph.microsoft.com/beta/users?top=999&filter=externalUserState eq 'Accepted' and userType eq 'Guest' and createdDateTime le ${min_user_age_date}&select=id,displayName,signInActivity,createdDateTime,mail" > users-${counter}.json
```

Comment:
could possibly remove this `userType eq 'Guest'` we do have some users who are members, (you can change it after inviting), it removes a couple of limitations in the tenant by doing that.